sbin/firehol

```diff
@@ -12465,7 +12404,7 @@ then
 		then
 			# RedHat
 			FIREHOL_AUTOSAVE="/etc/sysconfig/iptables"
-		elif [ -d "/var/lib/iptables" ]
+		elif [ -d "$LOCALSTATEDIR/lib/iptables" ]
```

Comment:
Agreed, I will put it back.

Actually I think the whole thing should be detected in configure then the logic can be removed from this script. But that is for a separate piece of work.